Generals/Code/GameEngine/Source/Common/Recorder.cpp

```diff
@@ -1079,10 +1079,16 @@ void RecorderClass::handleCRCMessage(UnsignedInt newCRC, Int playerIndex, Bool f
 			printf("CRC Mismatch in Frame %d\n", mismatchFrame);
 
 			// TheSuperHackers @tweak Pause the game on mismatch.
-			Bool pause = TRUE;
-			Bool pauseMusic = FALSE;
-			Bool pauseInput = FALSE;
-			TheGameLogic->setGamePaused(pause, pauseMusic, pauseInput);
+			if (TheMouse->getVisibility())
```

Comment:
This is not a good workaround. It is counter-intuitive and an unrelated check as well, because this is checking mouse visibility, not key input availability.

The better implementation here is to enable the input on setGamePaused, and then disable the input again after unpause. Basically improve the implementation of `setGamePaused`.